cpp/daal/src/algorithms/multiclassclassifier/multiclassclassifier_predict_votebased_impl.i

```diff
@@ -259,18 +259,24 @@ public:
 protected:
     Status getDataBlock(size_t startRow, size_t nRows, const NumericTable * a, NumericTablePtr & xTable) DAAL_C11_OVERRIDE
     {
-        _xRows.set(dynamic_cast<CSRNumericTableIface *>(const_cast<NumericTable *>(a)), startRow, nRows);
+        const bool toOneBaseRowIndices = true;
+        _xRows.set(dynamic_cast<CSRNumericTableIface *>(const_cast<NumericTable *>(a)), startRow, nRows, toOneBaseRowIndices);
         DAAL_CHECK_BLOCK_STATUS(_xRows);
+
+        algorithmFPType * const values = const_cast<algorithmFPType *>(_xRows.values());
+        size_t * const cols            = const_cast<size_t *>(_xRows.cols());
+        size_t * const rows            = const_cast<size_t *>(_xRows.rows());
         Status s;
-        xTable = CSRNumericTable::create(const_cast<algorithmFPType *>(_xRows.values()), _xRows.rows(), _xRows.cols(), a->getNumberOfColumns(), nRows,
-                                         CSRNumericTableIface::oneBased, &s);
+        xTable = CSRNumericTable::create(values, cols, rows, a->getNumberOfColumns(), nRows, CSRNumericTableIface::oneBased, &s);
         return s;
     }
 
 private:
     SubTaskVoteBasedCSR(size_t nClasses, size_t nRows, const SharedPtr<ClsType> & simplePrediction, bool & valid)
-        : super(nClasses, nRows, simplePrediction, valid)
+        : super(nClasses, nRows, simplePrediction, valid), _rowOffsets(nRows + 1)
     {}
+
+    TArrayScalable<size_t, cpu> _rowOffsets;
```

Comment:
You don't need _rowOffsets here anymore. They were moved to GetRowsCSR.

Reply:
Fixed